packages/celotool/geth_tests/src/lib/utils.ts

```diff
@@ -215,7 +216,13 @@ export async function killGeth() {
   await execCmd('pkill', ['-9', 'geth'], { silent: true })
 }
 
-function addStaticPeers(datadir: string, enodes: string[]) {
+export async function killInstance(instance: GethInstanceConfig) {
+  if (instance.pid) {
```

Comment:
What is this is not set?

Reply:
this should always be set when `startGeth` is called on an instance. If it is not set, the the instance isn't running and therefore killInstance should do nothing.